docs/api/commands/setcookie.mdx

```diff
@@ -38,6 +38,7 @@ Pass in an options object to change the default behavior of `cy.setCookie()`.
 | `log`      | `true`                                                         | Displays the command in the [Command log](/guides/core-concepts/cypress-app#Command-Log)                                                                                                                              |
 | `domain`   | Hostname of the current URL                                    | The domain the cookie is visible to                                                                                                                                                                                   |
 | `expiry`   | 20 years into the future                                       | When the cookie expires, specified in seconds since [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time).                                                                                                            |
+| `hostOnly` | `false`                                                        | Whether the cookie is a host only cookie   
```

Comment:
@AtofStryker What change is this doc update related to? I has assumed https://github.com/cypress-io/cypress/pull/25761, but we didn't add this option to the command or cookie typescript types. Also, looks like a TODO still in the code: https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/cookies.ts#L7

Reply:
pulled from https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/cookies/Cookie
```suggestion
| `hostOnly` | `false`                                                        | Whether the cookie is a host-only cookie, (i.e. the request's host must exactly match the domain of the cookie)
```

Reply:
While I was in the cookie deep dive I figured I could just add this in https://github.com/cypress-io/cypress/pull/25853 since it's pretty simple. I'll remove the todo in the PR. My guess is this change doesn't make it into the patch 😅